src/components/templates/Layout/Layout.tsx

```diff
@@ -16,14 +16,14 @@ export const Layout = ({
             data-testid='wrapper-div'
         >
             <div className='w-full'>
-                <header className='sticky top-0 z-30 w-full bg-universeBlue'>
+                <header className='sticky top-0 z-30 w-full bg-neutral-900'>
                     {navBar}
                 </header>
                 <main className='w-full'>{children}</main>
             </div>
             <footer
                 className={clsx('w-full', {
-                    'sticky bottom-0 z-30 bg-universeBlue':
+                    'sticky bottom-0 z-30 bg-neutral-900':
```

Comment:
@as472 I meant to replace universeBlue with neutral-900 because they were using the same hex code. Also, I can refer to this figma: https://www.figma.com/file/o2wjskNCdkt65haAgWPxo6/Responsive-Designs-(Post-Mainnet-Launch)?type=design&node-id=15895-114336&mode=dev which uses neutral 900

Reply:
@as472 Got it. I wasn't updating based off of the latest designs. Shall I keep things status quo? All my universeBlue -> neutral-900 changes are just to avoid the duplicate tailwind config colors.